src/development/docker-compose.yml

```
@@ -35,6 +35,16 @@ services:
       context: ./../Altinn.Platform/Altinn.Platform.PDF/
     extra_hosts:
       - "host.docker.internal:host-gateway"
+  altinn_pdf_service:
+    container_name: altinn-pdf-service
+    image: browserless/chrome:1-puppeteer-19.2.2
+    restart: always
```

Should probably hide this behind a `--profile` flag, so that before the official release it can be started with a special command.
```bash
docker compose --profile localtest --profile pdf up --build
```

```suggestion
    restart: always
    profiles:
        - pdf
```

---

thanks! didn't think of that!